elephant/test/test_pandas_bridge.py

```diff
@@ -235,23 +235,23 @@ def test__spiketrain_to_dataframe__parents_empty(self):
         self.assertEqual(keys, res8.columns.names)
 
         for value, level in zip(values, res0.columns.levels):
-            assert_index_equal(value, level)
+            assert_index_equal(value, level, False, False)
```

Comment:
(1) please can you add the keywords i.e. `exact=False, check_names=False` to make it clearer what is going on.
(2) is it necessary to have `exact=False`?